### PR TITLE
Unified build script

### PR DESCRIPTION
**Pull request by SinTh0r4s**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Do you need consistent player progressing (for example Thaumcraft)? -> Select a name
+# Do you need to test how your custom blocks interacts with a player that is not the owner? -> leave name empty
+developmentEnvironmentUserName = Developer
+
+# Define a source file of your project with:
+# public static final String VERSION = "GRADLETOKEN_VERSION";
+# The string's content will be replaced with your mods version when compiled. You should use this to specify your mod's
+# version in @Mod([...], version = VERSION, [...])
+# Leave these properties empty to skip individual token replacements
+replaceGradleTokenInFile = Tags.java
```
Comment (glowredman) [inline gradle.properties:29]: Shouldn't this be [LogisticsPipes.java](https://github.com/GTNewHorizons/LogisticsPipes/blob/9c72709d1bc71732594c03bcc655e74eb46c39d0/src/main/java/logisticspipes/LogisticsPipes.java#L163)?

**Review by glowredman — APPROVED** [review]
